extensions/ql-vscode/src/remote-queries/remote-queries-interface.ts

```diff
@@ -91,7 +91,7 @@ export class RemoteQueriesInterfaceManager {
       const { ctx } = this;
       const panel = (this.panel = Window.createWebviewPanel(
         'remoteQueriesView',
-        'Remote Query Results',
+        'CodeQL Query Results',
```

Comment:
According to the Figma designs, this should've been "CodeQL Query Results" anyway (to match the local results view).

Happy to change to "Variant Analysis Results" if people prefer!